polkadot-parachains/rockmine/src/constants.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod currency {
+	use node_primitives::Balance;
+
+	/// The existential deposit. Set to 1/10 of its parent Relay Chain.
+	pub const EXISTENTIAL_DEPOSIT: Balance = 100 * MILLICENTS;
+
+	pub const UNITS: Balance = 1_000_000_000_000;
+	pub const CENTS: Balance = UNITS / 100;
+	pub const MILLICENTS: Balance = CENTS / 1_000;
+	pub const GRAND: Balance = CENTS * 100_000;
+
+	pub const fn deposit(items: u32, bytes: u32) -> Balance {
+		// 1/10 of Rococo testnet
+		(items as Balance * 100 * CENTS + (bytes as Balance) * 5 * MILLICENTS) / 10
+	}
+}
+
+/// Fee-related.
+pub mod fee {
+	use frame_support::weights::{
+		constants::ExtrinsicBaseWeight, WeightToFeeCoefficient, WeightToFeeCoefficients,
+		WeightToFeePolynomial,
+	};
+	use node_primitives::Balance;
+	use smallvec::smallvec;
+	pub use sp_runtime::Perbill;
+
+	/// The block saturation level. Fees will be updates based on this value.
```

Comment:
```suggestion
	/// The block saturation level. Fees will be updated based on this value.
```

Reply:
Eagle sight 🦅 👀

Fixing it also in the rest of runtimes